src/plugins/data-inview/data-inview-en.hbs

```diff
@@ -42,11 +39,11 @@
 		</details>
 	</section>
 
-	<p>Different example text. Different example text. Different example text. Different example text. Different example text. Different example text. Different example text. Different example text.</p>
+	<p role="presentation">Different example text. Different example text. Different example text. Different example text. Different example text. Different example text. Different example text. Different example text.</p>
```

Comment:
I did some testing with NVDA and I will change them for ```role="img" aria-label=""```. I will update my PR tonight.